api/slack_test.go

```diff
@@ -0,0 +1,82 @@
+package api
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/slack-go/slack"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+type MockSlackClient struct {
+	mock.Mock
+}
+
+func (m *MockSlackClient) PostMessage(channelID string, options ...slack.MsgOption) (string, string, error) {
+	args := m.Called(channelID, options)
+	return args.String(0), args.String(1), args.Error(2)
+}
+
+func TestSendSlackMessagesSuccess(t *testing.T) {
+	// Create a mock Slack client
+	mockClient := new(MockSlackClient)
```

Comment:
Ah interesting I've never seen the `new()` function used. I've always opted for literals instead like `mockClient := MockSlackClient{}`. No need for change here, just found this interesting!